.gitignore

```diff
@@ -0,0 +1,2 @@
+.vscode
```

Comment:
I would recommend against putting workflow specific files into `.gitignore`; see http://stackoverflow.com/a/1753078/890 for tips on using `.git/info/exclude` to ignore user specific files.